speech/google/cloud/speech/result.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a Speech API Result."""
+
+from google.cloud.speech.transcript import Transcript
+
+
+class SpeechResult(object):
```

Comment:
My goal is to match the usage of the JSON version of the API to the gRPC version of the API so they're the same.

Also this pattern logically follows the other two methods(`sync_recognize`, `streaming_recognize`).